src/shared/components/annotation/OncoKB.tsx

```diff
@@ -78,7 +78,15 @@ export default class OncoKB extends React.Component<IOncoKbProps, {}>
             <span className={`${annotationStyles["annotation-item"]}`} />
         );
 
-        if (this.props.indicator)
+        if (this.props.indicator === null) {
+            // null means there is an error...
+            oncoKbContent = this.errorIcon();
```

Comment:
I think it would be better to add a prop `error:boolean` - it's a bit confusing this way.